README.md

```diff
@@ -88,24 +75,26 @@ attempt to mark the queue file as read-only and create a new file. The
 Receiver is programmed to read its current queue file until it reaches EOF
 and finds the file is read-only, at which point it deletes the file--it is
 the only reader--and moves on to the next.
-    
-If the Receiver is unable to keep up with the Senders then, oops, your disk will
-gradually fill up.
+
+`hopper::channel_with_max_bytes` takes a `max_disk_files` argument, defining the
+total number of overflow files that can exist concurrently. If all memory and
+disk buffers are full sends into the queue will fail but the error result is
```

Comment:
the sentence beginning `If all memory...` is difficult to read. the verbal noun `sends` can be confusing. i'd also strongly recommend breaking the error result subordinate clause into it's own sentence.

Reply:
how about `are full, sends into the queue fill fail. However, the error result is implemented such that the caller...`
This clarifies the noun-ness of sends by explicitly breaking the if clause into predicate and effect, and avoids the word "written" which can be ambiguous when used to mean 'how the code is written' while talking about a system that writes to disk.